apps/RawLogViewer/main_imports_exports.cpp

```diff
@@ -1688,8 +1689,8 @@ void xRawLogViewerFrame::OnMenuItemImportBremenDLRLog(wxCommandEvent& event)
 	std::string line;
 
 	mrpt::system::TTimeStamp cur_timestamp = mrpt::system::now();
-	const mrpt::system::TTimeStamp time_steps =
-		mrpt::system::secondsToTimestamp(0.25);  // why not? ;-)
+	const auto time_steps =
+		std::chrono::microseconds(static_cast<uint64_t>(0.25 * 1e6));
```

Comment:
This might be a bit convoluted. std::chrono::milliseconds(250) is much clearer.

Reply:
This one is crystal clear, yep!